Source/App/EbAppInputy4m.c

```diff
@@ -314,8 +314,18 @@ int32_t read_y4m_frame_delimiter(EbConfig_t *cfg) {
     fresult = fgets((char *)bufferY4Mheader, sizeof(bufferY4Mheader), cfg->inputFile);
 
     if (fresult == NULL) {
-        assert(feof(cfg->inputFile));
-        return EB_ErrorNone;
+        // Could be at end of file
+        // Rewind file and skip over header
+        fseek(cfg->inputFile, 0, SEEK_SET);
+        char buffer[YFM_HEADER_MAX];
+        fresult = fgets(buffer, sizeof(buffer), cfg->inputFile);
+
+        // try reading delimiter again
+        fresult = fgets((char *)bufferY4Mheader, sizeof(bufferY4Mheader), cfg->inputFile);
+        if (fresult == NULL) {
```

Comment:
btw, bufferY4Mheader naming is misleading, which is actually the frame header. `Each frame begins with the 5 bytes 'FRAME' followed by zero or more parameters each preceded by 0x20, ending with 0x0A. This is then followed by the raw bytes from each plane.`

Reply:
Agreed.. I can easily rename the variable